### PR TITLE
Change logging level logic, drop Error()

### PR DESCRIPTION
**Pull request by cortex**
This PR adds the following logic:
- HTTP status < 400 will be logged as Info
- HTTP status >=400 <500 will be logged as Info with the body as message
- HTTP status >= 500 will be logged as Error with the body as message

With this logic the Error() call is not necessary. This will break existing apps, the fix is to replace httplib.Error() with http.Error()

### CODE REVIEW

File: web.go [hunk 1]
```diff
@@ -88,20 +91,16 @@ func LogRequest(handler http.Handler) http.Handler {
...
+			case lw.status >= 400 && lw.status < 500:
+				lm.Info(fmt.Sprintf("%s\n%s", http.StatusText(lw.status), lw.body))
 			default:
 				lm.Error(fmt.Sprintf("%s\n%s", http.StatusText(lw.status), lw.body))
 			}
 		})
 }
 
-// Error wraps http.Error so we get the error message we return logged in the system
-func Error(w http.ResponseWriter, error string, code int) {
```
Comment (isaacbernat) [inline web.go:100]: Don't merge until I remove everywhere wrapp.Error is called!

File: web.go [hunk 2]
```diff
@@ -71,6 +71,9 @@ func Recover(handler http.Handler) http.Handler {
 }
 
 // LogRequest is a middleware that logs a request
+// HTTP status < 400 will be logged as Info
+// HTTP status <=400 <500 will be logged as Info with the body as message
```
Comment (eskilandreen) [inline web.go]: Don't we mean >=400 <500 ? Or in plain English, from 400 (inclusive) to 500 (exclusive)

Comment (cortex) [inline web.go]: yes =). The code is right, the comment is wrong.
